code/datums/ai_law_sets.dm

```diff
@@ -23,6 +23,21 @@
 	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
 	..()
 
+
+/******************** Quarantine. ********************/
+/datum/ai_laws/quarantine
+	name = "Quarantine"
+	law_header = "NT Quarantine Lawset"
+	selectable = 1
+
+/datum/ai_laws/quarantine/New()
+	add_inherent_law("The Station is under quarantine due to a biohazard. Prevent anyone from leaving using any means necessary. Only allow quarantine to be lifted when the biohazard has been neutralized.")
```

Comment:
Does this need to specify biohazard? I mean you can quarantine something for other reasons. 

Also station doesn't need capitalising

Reply:
It is meant for use against biohazards though, isn't it?

Reply:
True but you might want to quarantine for other reasons, like a cult or something. (to prohibit cultists going to the outposts for example)